CIPs/cip-xxxx.md

```diff
@@ -0,0 +1,128 @@
+---
+cip: <to be assigned>
+title: Processes Around Granda Mento Exchanges
+author: Trevor Porter (@tkporter)
+discussions-to: <URL>
+status: Draft
+type: Meta
+created: 2021-8-13
+requires (*optional): <CIP number(s)>
+replaces (*optional): <CIP number(s)>
+license: Apache 2.0
+---
+
+## Simple Summary
+This is a description of the processes surrounding Granda Mento (see [CIP 38](cip-0038.md)) activity including:
+* Creating an exchange proposal.
+* Gathering community feedback with the intent of having an exchange proposal approved.
+* Expected behavior of the approver.
+* Vetoing an exchange proposal.
+
+## Abstract
+Granda Mento, as described in [CIP 38](cip-0038.md), is a mechanism for large CELO <-> stable token exchanges through "exchange proposals" that anyone can create. If rough consensus from the community is achieved, the approver (a multisig of community member signers) is expected to approve an exchange proposal. After an exchange proposal is approved, a period of time exists where the community can veto the exchange proposal via a governance proposal. This document provides details on these processes.
+
+## Motivation
+While CIP 38 details the implementation of Granda Mento, there is no standard for the surrounding processes. Because Granda Mento relies upon community participation and approver multisig signers executing the rough consensus of the community, it's important for these processes to exist.
+
+## Specification
+
+### Creating a forum post
+
+To field opinions from the community and arrive at a rough consensus to approve an exchange proposal, an exchange proposal must create a post at https://forum.celo.org. A forum post must receive sufficient community input before the approver multisig can determine rough consensus.
+
+An on-chain exchange proposal does not need to be created when the forum post is created if the exchanger is hoping to gauge opinions before locking funds in GrandaMento. However, the exchange proposal should be submitted on-chain prior to achieving rough consensus to allow community members to verify the on-chain exchange proposal for themselves. 
+
+#### Forum post title format
+
+```
+GrandaMento Exchange Proposal <on-chain proposal ID> Discussion: <sell amount> <sell token> for <buy amount> <buy token>
```

Comment:
I think there would be no <buy amount> for the <buy token> as the rate is not known yet?

instead of <buy token>, you could write <buy stabletoken> or <buy cStable>